### ISSUE TITLE
Object context pooling

### ISSUE THREAD

**Issue by frk1993** [openseg-group/OCNet.pytorch#63]
Hi all,

Can someone describe the object comtext pooling moodule working principle? I dont understand what query, key and value transform functions are and how they work.

Thank you!

**Comment by PkuRainBow** [comment]
Please refer to the original [self-attention paper](https://papers.nips.cc/paper/7181-attention-is-all-you-need.pdf)

**Comment by frk1993** [comment]
Thank you for the response. I read the paper, you are building the correlation matrix and then use softmax like in the DANet approach ?

**Comment by PkuRainBow** [comment]
Exactly.
